src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Kiezkassen/Context/header.html

```diff
@@ -6,8 +6,13 @@
         <a
             href="{{processUrl | adhResourceUrl:'create_proposal'}}"
             class="button m-call-to-action"
-            data-ng-class="{'ng-disabled': !processOptions.POST && processOptions.loggedIn}">
+            data-ng-if="!(!processOptions.POST && processOptions.loggedIn)">
             {{ "TR__CREATE_PROPOSAL" | translate }}</a>
+        <span
+            class="button m-call-to-action"
+            data-ng-if="!processOptions.POST && processOptions.loggedIn"
+            data-ng-class="{'ng-disabled': true}">
```

Comment:
Just use `class="ng-disabled" in this case. `ngClass` is for conditional classes and has no further effects.